node/pkg/watchers/evm/connectors/batch_poller.go

```diff
@@ -70,6 +72,15 @@ func (b *BatchPollConnector) SubscribeForBlocks(ctx context.Context, errC chan e
 	innerErrSink := make(chan string, 10)
 	innerErrSub := b.errFeed.Subscribe(innerErrSink)
 
+	// Use the standard geth head sink to get latest blocks.
```

Comment:
Should we mention that the intent is to capture every block including rollback corrections? And that this should only be replaced with polling if the replacement code also sufficiently handles replaying rollback replacement blocks?